<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultPage</class>
 <widget class="QDialog" name="ResultPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>再来一局</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>游戏结束</string>
   </property>
  </widget>
  <widget class="QLabel" name="scoreLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lengthLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
